test/fixtures/templates/bad/resources/s3/bucket_policy.yaml

```diff
@@ -0,0 +1,114 @@
+---
+AWSTemplateFormatVersion: "2010-09-09"
+Conditions:
+  myCondition: !Equals [!Ref 'AWS::Region', 'us-east-1']
+Resources:
+  myBucket:
+    Type: AWS::S3::Bucket
+  BpBadActionWithBucketResource:
+    Type: AWS::S3::BucketPolicy
+    Properties:
+      Bucket: !Ref myBucket
+      PolicyDocument:
+        Statement:
+          - Action: s3:GetObject
```

Comment:
What about wilcard usages?

Like `s3:get*`?

Reply:
added a test for it to get this covered.